examples/window-app/efr32/BUILD.gn

```diff
@@ -84,13 +90,26 @@ if (silabs_board == "BRD4166A" || silabs_board == "BRD2601B" ||
   disable_lcd = true
 }
 
+defines = []
+
 # WiFi settings
 if (chip_enable_wifi) {
   # disabling LCD for MG24 for wifi
   if (silabs_board == "BRD4186C" || silabs_board == "BRD4187C") {
     show_qr_code = false
     disable_lcd = true
   }
+  if (chip_default_wifi_ssid != "") {
+    defines += [
+      "CHIP_ONNETWORK_PAIRING = 1",
```

Comment:
spacing in defines seems inconsistent here and below (and probably other places). Could you remove spaces before and after `=` for consistency? Also I am not sure how compiler arguments work when you have a space since `foo=bar` will be a single argument where as `foo =bar` will be 2 and `foo = bar` will be 3.